src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -820,7 +820,7 @@ public async Task ConnectTimeout_NotWrappedInMultipleTimeoutExceptions()
             Assert.Contains("ConnectTimeout", connectTimeoutException.Message);
 
             Assert.Null(connectTimeoutException.InnerException);
-            Assert.DoesNotContain("42", e.ToString());
+            Assert.DoesNotContain("HttpClient.Timeout", e.ToString());
```

Comment:
cc @MihaZupan it failed because some line number in the stacktrace contained 42 😄

Reply:
🤦‍♂️ Thanks for fixing this